### ISSUE TITLE
TDS Tooltip - Accessibility & Language issues

### ISSUE THREAD

**Issue by paultheva** [telus/tds-core#908]
## Description
- The `aria-label` inside the icon "Reveal additional information" does not seem to be translatable in French, we can add more text after it but not actually change it. Additionally, the Wave extension seems to flag the button as an error ("Empty label") because it doesn't see any content inside it.
- If possible, please translate "Reveal additional information" if it's used in a `/fr/` page to "Afficher l’information supplémentaire". If the "Empty label" issue is not a false flag, there needs to be an actual label inside of the button for screen readers, etc.

## Reproduction Steps
1. View the Tooltip component on a `/fr/` page.
2. `aria-label` remains in English.
3. Scan the page with Wave and the "Empty label" issue pops (might be a false flag?).

## Meta
- TDS component version: @tds/core-tooltip”: “^2.0.4
- Willing to develop solution: No
- Has workaround: No
- High impact: No

## Screenshots
![1](https://user-images.githubusercontent.com/8562738/51937905-c74b7f80-23d9-11e9-9188-292dcb63281a.png)

**Comment by theetrain** [comment]
Thanks for reporting this concern! I see we have hard-coded English copy when it should be overridable. This conversation is very similar to #931 and we are looking for a consistent method of handling accessible text in TDS.

**Comment by jraff** [comment]
@owestinTelus @mejsullivan

**Comment by marcod1419** [comment]
This has been addressed in [`Tooltip` 4.0.0](https://github.com/telus/tds-core/releases/tag/%40tds%2Fcore-tooltip%404.0.0) and [`StandaloneIcon` 1.1.2](https://github.com/telus/tds-core/releases/tag/%40tds%2Fcore-standalone-icon%401.1.2). Thanks for reporting!
